supersim.go

```diff
@@ -3,25 +3,30 @@ package supersim
 import (
 	"context"
 
+	"github.com/ethereum-optimism/supersim/anvil"
+
 	"github.com/ethereum/go-ethereum/log"
 )
 
 type Supersim struct {
 	log log.Logger
+
+	anvil *anvil.Anvil
 }
 
 func NewSupersim(log log.Logger) *Supersim {
-	return &Supersim{log}
+	anvil := anvil.New(log, &anvil.Config{ChainId: 10, Port: 9545})
+	return &Supersim{log, anvil}
 }
 
-func (s *Supersim) Start(ctx context.Context) error {
+func (s *Supersim) Start(_ context.Context) error {
 	s.log.Info("starting supersim")
-	return nil
+	return s.anvil.Start()
 }
 
-func (s *Supersim) Stop(ctx context.Context) error {
+func (s *Supersim) Stop(_ context.Context) error {
 	s.log.Info("stopping supersim")
-	return nil
+	return s.anvil.Stop()
 }
 
 func (s *Supersim) Stopped() bool {
```

Comment:
This should probably return `s.anvil.Stopped()` maybe instead of false, to reflect the status of the anvil instance. What do you think?

Reply:
ah yea that's right! switched the lifecycle a bit since the last draft and didn't update this. nice catch